ocp_resources/storage_profile.py

```diff
@@ -4,4 +4,33 @@
 
 
 class StorageProfile(Resource):
+    """
+    StorageProfile Object
+
+    Doc:
+    https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md
+    """
+
     api_group = Resource.ApiGroup.CDI_KUBEVIRT_IO
+
+    @property
+    def claim_property_sets(self):
+        return self.instance.status.get("claimPropertySets", {})
+
+    def first_claim_property_set_access_modes(self):
+        return self.claim_property_sets[0].get("accessModes")
+
+    def first_claim_property_set_volume_mode(self):
+        return self.claim_property_sets[0].get("volumeMode")
```

Comment:
if `claim_property_sets` is empty, this will fail


```
In [1]: x = []

In [2]: x[0]
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
Cell In[2], line 1
----> 1 x[0]

IndexError: list index out of range

```

Reply:
Yes, needs to be reverted to the old code.